descartes_core/include/descartes_core/robot_model.h

```diff
@@ -129,7 +129,12 @@ class RobotModel
    * @return                 [description]
    */
   virtual bool isValidMove(const std::vector<double> &from_joint_pose, const std::vector<double> &to_joint_pose,
-                           double dt) const = 0;
+                           double dt) const
+  {
+    return isValidMove(from_joint_pose.data(), to_joint_pose.data(), dt);
+  }
+
+  virtual bool isValidMove(const double* s, const double* f, double dt) const = 0;
```

Comment:
Won't this require existing classes to implement this form of the method?  Is there a way to make this backwards compatible?